src/ChunkSplitters.jl

```diff
@@ -9,30 +9,45 @@ export chunks, getchunk
 
 This function returns an iterable object that will split the *indices* of `array` into
 to `nchunks` chunks. `type` can be `:batch` or `:scatter`. It can be used to directly iterate
-over the chunks of a collection in a multi-threaded manner.
+over the chunks of a collection in a multi-threaded manner. 
 
```

Comment:
```suggestion
Returns an iterator that splits the *indices* of `array` into `nchunks`-many chunks.
The iterator can be used to process chunks of `array` one after another or in parallel (e.g. with `@threads`).

The optional argument `type` can be `:batch` (default) or `:scatter`.
```

Reply:
I was unable to commit this directly, so I copied it to a new commit.